packages/cli/src/cli/lib/skills-install.ts

```
@@ -0,0 +1,258 @@
+  description?: string;
+  /** Markdown body with the leading frontmatter block removed. */
+  body: string;
+}
+
+/**
+ * Parse a skill markdown document, extracting `name`/`description` from the
+ * leading `---` YAML frontmatter block (if any) and the body that follows.
+ * Tolerant by design: a document with no frontmatter yields `body === raw`.
+ */
+export function parseSkill(raw: string): ParsedSkill {
+  const normalized = raw.replace(/^﻿/, '');
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)